src/spells/RwaSpell.sol

```diff
@@ -28,6 +28,10 @@ import "dss-interfaces/dss/GemJoinAbstract.sol";
 import "dss-interfaces/dapp/DSTokenAbstract.sol";
 import "dss-interfaces/dss/ChainlogAbstract.sol";
 
+interface ERC20 {
```

Comment:
We should probably keep the `Like` suffix for interfaces to be consistent with the rest of the project.

```suggestion
import "dss-interfaces/dapp/DSTokenAbstract.sol";
import "dss-interfaces/dss/ChainlogAbstract.sol";

interface ERC20Like {
```